build.gradle

```diff
@@ -57,23 +57,33 @@ configurations {
 }
 
 dependencies {
-    implementation 'org.apache.kafka:kafka-clients:2.3.1'
-    implementation 'org.apache.kafka:kafka-streams:2.3.1'
+    implementation ('org.apache.kafka:kafka-clients:2.3.1') {
+        exclude group: 'org.scala-lang', module: 'scala-reflect'
+    }
+    implementation ('org.apache.kafka:kafka-streams:2.3.1') {
+        exclude group: 'org.scala-lang', module: 'scala-reflect'
+    }
     implementation 'org.apache.avro:avro:1.9.0'
     implementation 'com.google.code.gson:gson:2.8.5'
     implementation 'org.apache.commons:commons-configuration2:2.7'
     implementation 'commons-lang:commons-lang:2.6'
     implementation 'org.slf4j:slf4j-api:1.7.30'
     implementation 'net.sf.jopt-simple:jopt-simple:5.0.4'
-    testImplementation 'org.apache.kafka:kafka_2.12:2.3.1'
+    testImplementation('org.apache.kafka:kafka_2.12:2.3.1') {
+        exclude group: 'org.scala-lang', module: 'scala-reflect'
+    }
+    testImplementation group: 'org.scala-lang', name: 'scala-reflect', version: '2.12.11'
     testImplementation 'org.apache.zookeeper:zookeeper:3.5.5'
     testImplementation('io.netty:netty-all:4.1.43.Final') {
         force = true
     }
     testImplementation 'commons-io:commons-io:2.6'
     testImplementation 'junit:junit:4.12'
     testImplementation 'com.github.stefanbirkner:system-rules:1.19.0'
-    kafkaInMemory 'org.apache.kafka:kafka_2.12:2.3.1'
+    kafkaInMemory ('org.apache.kafka:kafka_2.12:2.3.1') {
+        exclude group: 'org.scala-lang', module: 'scala-reflect'
+    }
+    kafkaInMemory group: 'org.scala-lang', name: 'scala-reflect', version: '2.12.11'
```

Comment:
check version

Reply:
Discussed and finally will be use 2.12.11